release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -450,6 +450,43 @@ Updated boot images has been promoted to GA for Amazon Web Services (AWS) cluste
 [id="ocp-release-notes-monitoring_{context}"]
 === Monitoring
 
+The in-cluster monitoring stack for this release includes the following new and modified features. 
+
+[id="ocp-4-18-monitoring-updates-to-monitoring-stack-components-and-dependencies"]
+==== Updates to monitoring stack components and dependencies
+
+This release includes the following version updates for in-cluster monitoring stack components and dependencies:
+
+* Metrics Server to 0.7.2
+* Prometheus 2.55.1
+* Prometheus Operator to 0.78.1
+* Thanos to 0.36.1
+
+[id="ocp-4-18-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red{nbsp}Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* With this update, you can create user workload alerting and recording rules that query multiple projects at the same time.
+
+[id="ocp-4-18-monitoring-added-scrape-and-evaluation-intervals-for-uwm-prometheus"]
+==== Added scrape and evaluation intervals for user workload monitoring Prometheus
+
+With this update, you can configure the intervals between consecutive scrapes and between rule evaluations for user workload monitoring Prometheus.
+
+[id="ocp-4-18-monitoring-early-validation-for-configurations-in-monitoring-config-maps"]
+==== Added early validation for the monitoring configurations in monitoring config maps
+
+This update introduces early validation for changes to monitoring configurations in `cluster-monitoring-config` and `user-workload-monitoring-config` config maps to provide shorter feedback loops and enhance user experience.
+
+[id="ocp-4-18-monitoring-configure-cross-project-alerting-rules-for-uwm"]
+==== Added the proxy environment variables to Alertmanager containers
+
+With this update, Alertmanager uses the proxy environment variables, therefore, you can enable proxying for all alert receivers.
```

Comment:
Maybe clarify further that you can set `proxy_from_environment: true` in your alert receivers to leverage the global HTTP proxy configuration?